framework/src/utils/MooseAppCoordTransform.C

```diff
@@ -204,6 +215,7 @@ MooseAppCoordTransform::validParams()
       "coord_type", coord_types, "Type of the coordinate system per block param");
   params.addParam<MooseEnum>(
       "rz_coord_axis", rz_coord_axis, "The rotation axis (X | Y) for axisymetric coordinates");
+  params.addParam<Point>("translation", "Translation vector. Override by the app position");
```

Comment:
this is up for deletion. It would be a way to define the position in the subapp input file. 

It s not compatible with the positions argument of the MultiApp
It also only works with run_in_position right now

Reply:
I would turn the question around and ask if you have interest in having this? On the one hand, it would nice to have some symmetry with the other types of transform. On the other if there's no feature advantage to this over `positions`, then I don't care to have multiple ways to do the same thing generally

Reply:
not really. i am doing this for symmetry

Reply:
let s delete it. My desire for symmetry isnt worth handling an extra parameter